### ISSUE TITLE
Vagrant error writing to Xenial log file in root directory

### ISSUE THREAD

**Issue by ghost** [clickontyler/VirtualHostX-Issues#76]
VirtualBox starts fine the first time you install it, any changes requiring the virtualbox to be rebooted or simply shut down VirtualHostX and open it again, won't start the VirtualBox but spit out errors.

Re-formated the computer, fresh install of High Sierra and only installed VirtualHostX, exact same bug, exact same error.

```
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "cb7664f7-ee9e-4c37-9a57-3ec083f58afe", "--type", "headless"]

Stderr: VBoxManage: error: RawFile#0 failed to create the raw output file /ubuntu-xenial-16.04-cloudimg-console.log (VERR_ACCESS_DENIED)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

Don't buy this crap, it's not working, at all.

**Comment by tylerhall** [comment]
What you're seeing is an error being thrown by Vagrant - a 3rd party app that VHX relies on. I've had two other customers email in with this same problem, and I'm investigating it. No solution yet.

**Comment by ghost** [comment]
I switched to Mamp Pro due to the lack of response here. No longer urgent / required for me.

**Comment by tylerhall** [comment]
This bug has been fixed in v7.4.1. The solution was to simply turn off the virtual serial port that was trying to write the log file to a root owned location.
